Source/diablo.cpp

```diff
@@ -507,14 +507,12 @@ static void SaveOptions()
 	setIniInt("Audio", "Channels", sgOptions.Audio.nChannels);
 	setIniInt("Audio", "Buffer Size", sgOptions.Audio.nBufferSize);
 	setIniInt("Audio", "Resampling Quality", sgOptions.Audio.nResamplingQuality);
-#ifndef __vita__
 	setIniInt("Graphics", "Width", sgOptions.Graphics.nWidth);
 	setIniInt("Graphics", "Height", sgOptions.Graphics.nHeight);
-#endif
-	setIniInt("Graphics", "Fullscreen", sgOptions.Graphics.bFullscreen);
 #ifndef __vita__
-	setIniInt("Graphics", "Upscale", sgOptions.Graphics.bUpscale);
+	setIniInt("Graphics", "Fullscreen", sgOptions.Graphics.bFullscreen);
 #endif
+	setIniInt("Graphics", "Upscale", sgOptions.Graphics.bUpscale);

```

Comment:
Looks like this should only be exposed when `#if !defined(USE_SDL1)` like in loadOptions